aether-kernel/aether/kernel/api/utils.py

```diff
@@ -606,40 +606,45 @@ def extract_create_entities(submission_payload, mapping_definition, schemas):
 
 
 def run_entity_extraction(submission):
-    # Get the mapping definition from the submission (submission.mapping.definition):
-    mapping_definition = submission.mapping.definition
-    # Get the primary key of the projectschema
-    # entity_pks = list(mapping_definition['entities'].values())
-    entity_ps_ids = mapping_definition.get('entities')
-    # Save submission and exit early if mapping does not specify any entities.
-    if not entity_ps_ids:
-        submission.save()
-        return
-    # Get the schema of the projectschema
-    project_schemas = {
-        name: models.ProjectSchema.objects.get(pk=_id) for name, _id in
-        entity_ps_ids.items()
-    }
-    schemas = {
-        name: ps.schema.definition for name, ps in
-        project_schemas.items()
-    }
-    submission.save()
-    _, entities = extract_create_entities(
-        submission_payload=submission.payload,
-        mapping_definition=mapping_definition,
-        schemas=schemas,
-    )
-    for entity in entities:
-        projectschema_name = entity.projectschema_name
-        projectschema = project_schemas[projectschema_name]
-        entity_instance = models.Entity(
-            payload=entity.payload,
-            status=entity.status,
-            projectschema=projectschema,
-            submission=submission,
-        )
-        entity_instance.save()
+    # Extract entity for each mapping in the submission.mappingset
+    mappings = models.Mapping.objects.filter(mappingset=submission.mappingset)
+    for mapping in mappings:
+        if mapping.is_active:
+            mapping_definition = mapping.definition
+            # Get the primary key of the projectschema
+            # entity_pks = list(mapping_definition['entities'].values())
+            entity_ps_ids = mapping_definition.get('entities')
+            # Save submission and exit early if mapping does not specify any entities.
+            if not entity_ps_ids:
+                submission.save()
+                return
+            # Get the schema of the projectschema
+            project_schemas = {
+                name: models.ProjectSchema.objects.get(pk=_id) for name, _id in
+                entity_ps_ids.items()
+            }
+            schemas = {
+                name: ps.schema.definition for name, ps in
+                project_schemas.items()
+            }
```

Comment:
Could you reformat these lines?
Instead of 
```python
{
    name: func(value) for value in 
    array
}
```
it's more readable like this
```python
{
    name: func(value)
    for value in array
}
```